asyncapi/quickbooks/Package.md

```diff
@@ -0,0 +1,18 @@
+Listen to [QuickBook Webhook](https://developer.intuit.com/app/developer/qbo/docs/develop/webhooks) from Ballerina
+
+## Package overview
+The `ballerinax/trigger.quickbook` is a [Ballerina](https://ballerina.io/) trigger for QuickBook.
```

Comment:
Let's use the same package name used in the Ballerina.toml. i.e. `ballerinax/trigger.quickbooks`